### PR TITLE
Add cargo test dummy_issuer to CI

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Do not add breaking changes to dummy_issuer accidentally.

In this PR, I add the integration tests of the dummy_issuer to the CI.

# Changes

* New workflow "Rust" to build the wasm and execute the tests.

# Tests

Already running in the PR.

# Todos

- [ ] Add entry to changelog (if necessary). Not necessary.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@frederikrothenberger please review

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust: [1.77.2]
```
Comment (frederikrothenberger) [inline .github/workflows/rust.yml:17]: I don't think we need support to test multiple rust versions, do we? What about moving that up to `env`?

**Review by frederikrothenberger — APPROVED** [review]
Maybe we should rename the CI pipeline for the relying party? It's now called "CI Checks" but is only one of four pipelines...

File: .github/workflows/rust.yml (hunk 1 above)
Comment (lmuntaner) [reply .github/workflows/rust.yml]: Good point. I removed it. I believe that we don't even need it because we have the rust-toolchain file

**Pull request merged**